regulations/templates/regulations/layers/table.html

```diff
@@ -2,7 +2,7 @@
     Used to turn markdown tables into HTML tables
 {% endcomment %}
 <div class="table-wrap">
-    <table>
+    <table width="100%">
```

Comment:
Yea - this (maddeningly) is needed.  If not present, the plugin does not compute and actual width for the scroll container - which is needed to facilitate scrolling via the header area.  This is mentioned in several threads in the Datatables forum.   When you say comment - do you want meek to comment in table.html?